makefile

```diff
@@ -402,6 +403,8 @@ tests-e2e: build-armadactl build-docker-no-lookout tests-e2e-setup
 	$(GO_TEST_CMD) go test -v ./e2e/armadactl_test/... -count=1 2>&1 | tee test_reports/e2e_armadactl.txt
 	$(GO_TEST_CMD) go test -v ./e2e/basic_test/... -count=1 2>&1 | tee test_reports/e2e_basic.txt
 	$(GO_TEST_CMD) go test -v ./e2e/pulsar_test/... -count=1 2>&1 | tee test_reports/e2e_pulsar.txt
+	docker run -v${PWD}/client/python:/code -e ARMADA_SERVER=server -e ARMADA_PORT=50051 --entrypoint python3 --network=kind armada-python-client-builder:latest -m pytest -v -s /code/tests/integration/test_no_auth.py
```

Comment:
If these aren't also added to `tests-e2e-no-setup`, they will not run in CI. In fact, they are not currently running in the circleci job triggered by the PR and so haven't been tested yet AFAICT.

Reply:
It's clearly in tests-e2e-no-setup above. I have no idea why I couldn't find it before Kevin pointed me at it.